chef_master/source/config_rb_supermarket.rst

```diff
@@ -5,84 +5,75 @@ supermarket.rb Settings
 
 .. tag config_rb_supermarket_summary
 
-The supermarket.rb file contains all of the non-default configuration settings used by the Chef Supermarket. (The default settings are built-in to the Chef Supermarket configuration and should only be added to the supermarket.rb file to apply non-default values.) These configuration settings are processed when the ``supermarket-ctl reconfigure`` command is run, such as immediately after setting up Chef Supermarket or after making a change to the underlying configuration settings after the server has been deployed. The supermarket.rb file is a Ruby file, which means that conditional statements can be used in the configuration file.
+The supermarket.rb file contains all of the non-default configuration settings used by the Chef Supermarket. The default settings are built-in to the Chef Supermarket configuration, and should only be added to the supermarket.rb file to apply non-default values. These configuration settings are processed when the ``supermarket-ctl reconfigure`` command is run. The supermarket.rb file is a Ruby file, which means that conditional statements can be used in the configuration file.
 
 .. end_tag
 
-.. note:: The supermarket.rb file does not exist by default. To modify the settings for the Chef server, create a file named ``supermarket.rb`` in the ``/etc/supermarket/`` directory.
+.. note:: The ``supermarket.rb`` file does not exist by default. To modify the settings for the Supermarket server, create a file named ``supermarket.rb`` in the ``/etc/supermarket/`` directory.
 
 Settings
 =====================================================
-The following settings are may be configured in the supermarket.rb file.
+The following settings are available in the ``supermarket.rb`` file.
 
-.. note:: When changes are made to the chef-server.rb file the Chef server must be reconfigured by running the ``supermarket-ctl reconfigure`` command.
+.. note:: You must run ``supermarket-ctl reconfigure`` to apply any changes made in the ``supermarket.rb`` file.
 
 General
 -----------------------------------------------------
 This configuration file has the following general settings:
 
 ``default['enterprise']['name']``
-   Default value: ``'supermarket'``.
+   The Chef Automate enterprise name. Default value: ``'supermarket'``.
```

Comment:
This `enterprise` bit isn't related to Automate. This is an attribute used by the `enterprise` cookbook.

Reply:
heh yeah, in retrospect that makes sense given the default value.